app/controllers/agenda/agendaitems/agendaitem/index.js

```diff
@@ -24,6 +24,7 @@ export default class IndexAgendaitemAgendaitemsAgendaController extends Controll
   @tracked decisionActivity;
 
   @tracked isEditingAgendaItemTitles = false;
+  //TODO: add isClosed
```

Comment:
Any reason why this TODO is here? Don't see any use for `isFinal`/`isClosed` in this controller. I'll remove this